pages/documentation/developer/api_guide/single-line-diagram_svg-writing.md

```diff
@@ -0,0 +1,214 @@
+---
+layout: default
+---
+
+# Single Line Diagram - SVG Writing
+
+The `com.powsybl.sld.svg.DefaultSVGWriter` class of [powsybl-single-line-diagram](../repositories/powsybl-single-line-diagram.md) is an implementation of `SVGWriter` interface which allows to generate SVG files representing single line diagrams of a substation, a voltage level or a zone. 
+We are showing in this guide how to create some single line diagrams
+- first from a test network,
+- then from a CGMES file.
+
+*Note: the 1.8.0-SNAPSHOT version of powsybl-single-line-diagram was used to generate the SVGs of this guide. Because of that, you might experience some differences with the examples shown, which will be corrected by 1.8.0 release (expected: 2020-11-26).*
+
+
+## Prerequisites
+
+### Maven dependencies
+First of all, we need to add some Maven dependencies in our `pom.xml` file:
+
+```xml
+<dependencies>
+    <dependency>
+        <groupId>com.powsybl</groupId>
+        <artifactId>powsybl-single-line-diagram-core</artifactId>
+        <version>${powsybl.sld.version}</version>
+    </dependency>
+    <dependency>
+        <groupId>com.powsybl</groupId>
+        <artifactId>powsybl-iidm-impl</artifactId>
+        <version>${powsybl.core.version}</version>
+    </dependency>
+    <dependency>
+        <groupId>com.powsybl</groupId>
+        <artifactId>powsybl-iidm-test</artifactId>
+        <version>${powsybl.core.version}</version>
+    </dependency>
+    <dependency>
+        <groupId>com.powsybl</groupId>
+        <artifactId>powsybl-config-test</artifactId>
+        <version>${powsybl.core.version}</version>
+    </dependency>
+    <dependency>
+        <groupId>com.powsybl</groupId>
+        <artifactId>powsybl-cgmes-conversion</artifactId>
+        <version>${powsybl.core.version}</version>
+    </dependency>
+    <dependency>
+        <groupId>com.powsybl</groupId>
+        <artifactId>powsybl-triple-store-impl-rdf4j</artifactId>
+        <version>${powsybl.core.version}</version>
+    </dependency>
+    <dependency>
+        <groupId>org.slf4j</groupId>
+        <artifactId>slf4j-simple</artifactId>
+        <version>${slf4j.version}</version>
+    </dependency>
+</dependencies>
+
+<properties>
+    <powsybl.sld.version>1.7.2</powsybl.sld.version>
```

Comment:
This is not consistent with the sentence above: you say that this guide depends on v1.8.0

You should also fix the powsybl.core version to v3.8.0